frontend/src/streamlist/StreamsTableWrapper.jsx

```diff
@@ -17,8 +17,8 @@ export default class StreamsTableWrapper extends React.Component {
     this.dataSource
       .getAllStreams()
       .then(this.handleStreamsChange)
-      .catch(() => {
-        this.handleStreamsChange(getAllStreams());
+      .catch((error) => {
+        snackbar("error", `Failed to fetch stream data: ${error.message}`);
```

Comment:
suggesting use of `streams` instead of `stream data` to be consistent with "Failed to fetch senders/receivers" messages

Reply:
cc0b92d